acvm/src/compiler/transformers/fallback.rs

```diff
@@ -22,9 +22,8 @@ impl FallbackTransformer {
 
         for opcode in acir.opcodes {
             let bb_func_call = match &opcode {
-                Opcode::Arithmetic(_) | Opcode::Directive(_) => {
-                    // If it is not a black box function, then it is a directive or
-                    // an arithmetic expression which are always supported
+                Opcode::Arithmetic(_) | Opcode::Directive(_) | Opcode::Block(_, _) => {
```

Comment:
It seems we are adding a separate opcode, why can it not fit under the category of the ones we already have?

Reply:
This one is for emulating memory access. The idea is that a backend could implement it more efficiently.